stable/nginx-ingress/templates/controller-prometheusrules.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.controller.metrics.enabled .Values.controller.metrics.prometheusRule.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "nginx-ingress.controller.fullname" . }}
+  {{- if .Values.controller.metrics.prometheusRule.namespace }}
+  namespace: {{ .Values.controller.metrics.prometheusRule.namespace }}
```

Comment:
namespace is handled by helm and doesn't need to be included in our templates.

Reply:
Same as above, check templates/controller-servicemonitor.yaml ( https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/controller-servicemonitor.yaml#L6-L8 )